lib/concurrent/agent.rb

```diff
@@ -102,24 +104,24 @@ def validate(&block)
     alias_method :validate_with, :validate
     alias_method :validates_with, :validate
 
-    # Update the current value with the result of the given block operation,
+    # Update the current value with the result of the given block fast,
     # block should not do blocking calls, use #post_off for blocking calls
     #
-    # @yield the operation to be performed with the current value in order to calculate
+    # @yield the fast to be performed with the current value in order to calculate
     #   the new value
     # @yieldparam [Object] value the current value
     # @yieldreturn [Object] the new value
     # @return [true, nil] nil when no block is given
     def post(&block)
-      post_on(@task_executor, &block)
+      post_on(@io_executor, &block)
```

Comment:
This is the other way around in clojure. Default is fast, and post_off is for io. I think we should have sam behavior.